optaplanner-examples/src/test/java/org/optaplanner/examples/tsp/persistence/LowEntityCountTest.java

```diff
@@ -0,0 +1,62 @@
+package org.optaplanner.examples.tsp.persistence;
+
+import org.assertj.core.api.Assertions;
+import org.junit.Test;
+import org.optaplanner.core.api.solver.Solver;
+import org.optaplanner.examples.tsp.app.TspApp;
+import org.optaplanner.examples.tsp.domain.TspSolution;
+
+public class LowEntityCountTest {
+
+    public static final long LOW_ENTITY_SECONDS_TERMINATION = 1;
+    public static final String PATH_TO_NEARBY_SELECTION_XML = "org/optaplanner/examples/tsp/lowentitycount/nearbySelection.xml";
+    private TspApp app = new TspApp();
+
+    @Test
+    public void zeroEntity() {
+        Solver<TspSolution> tspSolutionSolver = app.createSolver(LOW_ENTITY_SECONDS_TERMINATION);
+        TspSolution problem = app.getUnsolvedProblem("oneDomicileOnly.xml");
+        Assertions.assertThatIllegalStateException().isThrownBy(() -> tspSolutionSolver.solve(problem)).
```

Comment:
Please add static imports for `assert*()` methods. We generally discourage using static imports but for tests it is actually recommended as it improves code legibility. The reason why we can afford statically imported methods in tests is because in that context the reader usually knows that things like `assertThat()`, `assertWhatever()`, `mock()`, `verify()` etc. are testing framework methods.